### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by achevali**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -289,7 +290,30 @@ java -Xmx8g -jar <path_to_hellbender_protected_jar>  PerformSegmentation  -S <sa
...
+- segmented coverage file (seg) -- from step 3.
+- sample name
+
+###### Outputs
+- beforeAfterTangentLimPlot (png) -- Output before_after tangent normalization plot up to copy-ratio 4 
+- beforeAfterTangentPlot (png) -- Output before_after tangent normalization plot 
+- fullGenomePlot (png) -- Full genome plot after tangent normalization
+- preQc (txt) -- median absolute differences of targets before normalization
+- postQc (txt) -- median absolute differences of targets after normalization
+- dQc (txt) -- differance in median absolute differences of targets before and after normalization
```
Comment (vdauwera) [inline README.md]: differance -> difference

File: README.md [hunk 2]
```diff
@@ -289,7 +290,30 @@ java -Xmx8g -jar <path_to_hellbender_protected_jar>  PerformSegmentation  -S <sa
...
+- normalized coverage file (tsv) -- from step 2.
+- pre-normalized coverage file (tsv) -- from step 2.
+- segmented coverage file (seg) -- from step 3.
+- sample name
+
+###### Outputs
+- beforeAfterTangentLimPlot (png) -- Output before_after tangent normalization plot up to copy-ratio 4 
+- beforeAfterTangentPlot (png) -- Output before_after tangent normalization plot 
+- fullGenomePlot (png) -- Full genome plot after tangent normalization
+- preQc (txt) -- median absolute differences of targets before normalization
```
Comment (vdauwera) [inline README.md]: Inconsistent capitalization of the descriptions

File: README.md [hunk 3]
```diff
@@ -289,7 +290,30 @@ java -Xmx8g -jar <path_to_hellbender_protected_jar>  PerformSegmentation  -S <sa
...
 
 
-##### Step 4. Call segments
+##### Step 4. Plot coverage profile
+
+###### Inputs
+- normalized coverage file (tsv) -- from step 2.
+- pre-normalized coverage file (tsv) -- from step 2.
+- segmented coverage file (seg) -- from step 3.
+- sample name
```
Comment (vdauwera) [inline README.md]: Any constraints on the sample name in terms of characters? (eg whitespace, special chars etc) Does it need to be passed in double quotes?

**Comment by vdauwera** [comment]
Just a couple of petty/pedantic corrections and a question about constraints on the sample name composition.

**Comment by achevali** [comment]
Updates for comments, back to you @vdauwera

**Comment by vdauwera** [comment]
Copacetic -- squash and merge at will

**Pull request merged**